modules/vms/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "instance_id" {
```

Comment:
Don't you wanna output the Public/Private IP from the created EC2 instance?

Reply:
yes it'll be handy in case user wants to ssh into the vm, thanks